package.json

```diff
@@ -32,14 +35,20 @@
     "@commitlint/cli": "^18.0.0",
     "@commitlint/config-conventional": "^18.0.0",
     "@contentful/eslint-config-extension": "0.4.3",
+    "@typescript-eslint/eslint-plugin": "^4.10.0",
+    "@typescript-eslint/parser": "^4.10.0",
     "commitizen": "4.3.0",
     "concurrently": "8.2.2",
+    "eslint": "^7.32.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-import-helpers": "^1.2.1",
+    "eslint-plugin-react": "7.33.2",
+    "eslint-plugin-react-hooks": "4.6.0",
     "husky": "^8.0.3",
     "lerna": "^8.0.0",
     "lint-staged": "^15.0.1",
-    "prettier": "2.8.8"
+    "prettier": "2.8.8",
+    "typescript": "5.1.3"
```

Comment:
why are you not using the latest version of typescript here?

Reply:
@aodhagan-cf we get [this error](https://app.circleci.com/pipelines/github/contentful/live-preview/1754/workflows/f7dd1238-4770-4d17-bcc5-7d3aa092c72b/jobs/3732) when we use the latest version & the fixes I saw online weren't working